BUILD.md

```diff
@@ -14,6 +14,13 @@ up builds. The minimum set of dependencies is:
 All other dependencies are optional and can be disabled while configuring the
 build (in `meson setup` step).
 
+## General notes
+
+Meson allows maintaining different build configurations for a number of build
+targets in their own separate subdirectories. E.g. you can keep your debug and
```

Comment:
Maybe a bit simpler: `It's possible to have several Meson build configurations using  subdirectories, for example, you can ..`


`Meson  subdirectories.`

Reply:
Thanks, that reads a lot better :)